InstabugSample/e2e/firstTest.e2e.js

```diff
@@ -1,8 +1,12 @@
 describe('Example', () => {
-  beforeEach(async () => {
+  beforeAll(async () => {
```

Comment:
Any reason behind this change?

Ideally, we'd want to restart the app before each test to make sure the native components are reset, and since `reloadReactNative` only reloads the JS bundle, we can't rely on it.